README.rst

```diff
@@ -692,6 +693,8 @@ to the default list of symfony-level fixers:
         ->finder($finder)
     ;
 
+NOTE: `exclude` will work only for directories, so if you need to exclude file, try `notPath`.
```

Comment:
maybe add `.gitignore` style matching? it is more stright forward to write and understand:

```
src/**.php
!src/Subdir
*.php
```